capstone/src/main/java/com/google/univiz/api/dataviz/CollegeStats.java

```diff
@@ -0,0 +1,46 @@
+package com.google.univiz.api.dataviz;
```

Comment:
After #77, we won't have dataviz package anymore. Make sure to move this into the right place i.e. api.representation.